doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -227,6 +227,10 @@ nRF9160 samples
         This includes several new fields in the PVT notification and a command to query the expiry times of assistance data.
       * Support for the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_STORAGE_PARTITION` option.
 
+    * Updated:
+
+      * When :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_MEDIUM_UART` is enabled, modem traces will be disabled before disabling UART1. They will be re-enabled when re-enabling UART1.
```

Comment:
```suggestion
      * When the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_MEDIUM_UART` option is enabled, modem traces are also disabled if UART1 is disabled.
        When you enable UART1 again, the modem traces are re-enabled.
```
Not at all sure I managed to keep the idea right. Anyway, the original seems very strange.

Reply:
I think the idea is mostly preserved. The UART1 can either be re-enabled on a timer expiry or by the user pressing a button. So the 'you' in second sentence may not be correct. Can I write the following?

`When the UART1 is re-enabled (either after timer expiry or button press), the modem traces are also re-enabled.`

Reply:
Thanks 👍  will fix